addon/components/paper-item.js

```diff
@@ -82,10 +82,14 @@ export default Component.extend(RippleMixin, ProxyMixin, {
         let el = component.$();
         set(this, 'mouseActive', false);
         el.on('mousedown', ()=> {
-          set(this, 'mouseActive', true);
-          run.later(()=> {
-            set(this, 'mouseActive', false);
-          }, 100);
+          if (!get(component, 'isDestroyed') && !get(component, 'isDestroying')) {
```

Comment:
So I did a bit of research and I couldn't come to a conclusive answer. The material design spec only states "Ink ripples confirm user input by immediately expanding outward from the point of touch"

On http://inbox.google.com, as well as the android and iOS apps, they don't use any ripple effect, anywhere. 

On Android, in the settings menu, the ripple effect does expand the entire equivalent of a `paper-item` but only if the right item is a toggle, such as the wifi menu option.

In the Sheets app, clicking on a card does produce a ripple effect, but clicking on the `more vert` icon in the card, only ripples the icon, and not the entire card.

I will fix the PR to any way you are happy. I guess there are two cases, a toggle makes sense to ripple the entire `paper-item` but a delete and a `vert more` does not.

What do you guys, think?